.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
         - CMD="mvn install --projects jmh-tests,performance-tests --also-make --activate-profiles all -DskipTests=true --batch-mode --show-version --errors"
 
 script:
-- if [[ "$TRAVIS_EVENT_TYPE" == "cron" && DESC="unit tests" ]];
+  if [[ "$TRAVIS_EVENT_TYPE" == "cron" && "$DESC" == "unit tests" ]];
   then mvn install --batch-mode --show-version --errors sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=eclipse-collections ;
```

Comment:
@motlin `TRAVIS_EVENT_TYPE` indicates how the build was triggered - see https://docs.travis-ci.com/user/environment-variables/ , `cron` is about https://docs.travis-ci.com/user/cron-jobs/ , so that analysis runs once a day according to your setup of Travis in order to respect "fair use" of a free SonarQube.com service - see https://about.sonarqube.com/